example/docker-compose.yml

```diff
@@ -49,10 +46,13 @@ services:
       - mysql
     depends_on:
       - mysql
-    restart: always
+      - wikibase
+    restart: unless-stopped
     volumes:
-      - shared:/shared/:ro
       - ./jobrunner-entrypoint.sh:/jobrunner-entrypoint.sh
+      - shared-localsettings:/var/shared-localsettings
```

Comment:
notably this swap now allow the jobrunner read/write access to the settings file that was previously read only

Reply:
I can add :ro back!